src/components/Skeleton/index.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+import styles from './styles.module.scss';
+
+const SkeletonBase = () => {
+  return <div className={styles.skeleton} />;
```

Comment:
This should accept a `className` prop so that the style can be adjusted more easily.

Reply:
Yep, I did not add that prop yet because I did not need to inject a className. We might also want to have other props.